### PR TITLE
[Bulk] Validation: Fix validation issues 4/25

### PR DESCRIPTION
**Pull request by RedZephyr13**
This PR fixes 89 validation issues.

### CODE REVIEW

File: docs-ref-services/preview/messaging-eventgrid-cloudnative-cloudevents-readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Cloud Native Cloud Event client library for Java
 keywords: Azure, java, SDK, API, azure-messaging-eventgrid-cloudnative-cloudevents, eventgrid
+description: Azure Cloud Native Cloud Event client library for Java, can be used to enable publishing the Cloud Native Computing Foundation using the Azure Event Grid library.
```
Comment (Roudabeh1348) [inline docs-ref-services/preview/messaging-eventgrid-cloudnative-cloudevents-readme.md:4]: delete the apostrophe between java and can

File: docs-ref-services/preview/messaging-eventgrid-readme.md [hunk 2]
```diff
@@ -401,17 +400,18 @@ byte[] dataInBytes = eventData.toBytes();
 ```  
 
 #### Deserialize system event data from `CloudEvent` or `EventGridEvent`
-An event that is sent to a [System Topic](https://docs.microsoft.com/azure/event-grid/system-topics) is called a
+An event that is sent to a [System Topic](/azure/event-grid/system-topics) is called a
```
Comment (Roudabeh1348) [inline docs-ref-services/preview/messaging-eventgrid-readme.md:403]: looks like there is break between line 403 and 404

Comment (RedZephyr13) [reply docs-ref-services/preview/messaging-eventgrid-readme.md:403]: Nice catch!

**Comment by RedZephyr13** [comment]
Global effort to fix build validation errors

@sima-zhu

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request closed without merging**